src/app/services/core-event-handlers.service.ts

```diff
@@ -89,12 +90,18 @@ export class CoreEventHandlersService {
     this.formSubmit.actions$
       .pipe(
       )
-      .subscribe(({ action }) => {
-        // console.log(`process action ${action}`);
+      .subscribe(({action, payload}) => {         
```

Comment:
This change is required in order to test the export adapter functionality. 
The process of installing an adapter via the standard, core, flows felt impossible to achieve. Because the process relied on local storage and local index db, most of the steps involved in setting up the tests did not reliably work.

Reply:
This has now been removed. New global settings flag introduced.